core/src/main/java/jenkins/security/ResourceDomainConfiguration.java

```diff
@@ -82,7 +82,7 @@ public ResourceDomainConfiguration() {
     @Restricted(NoExternalUse.class)
     @POST
     public FormValidation doCheckUrl(@QueryParameter("url") String resourceRootUrlString) {
-        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+        Jenkins.get().checkPermission(Jenkins.CONFIGURE);
```

Comment:
Revert, this seems like an admin thing.

Reply:
This method is used when the page "/configure" is opened, so if we want that some part of "Configure System" can be allowed to the CONFIGURE permission we have to allow it or move the call. I am looking which part of the "Configure system" page need that call.

Reply:
This ought to be moved to `/configureSecurity/` I think (change the category on the `GlobalConfiguration`).

Reply:
changed to ADMINISTER, I still put in the todolist to look to move that check because the "/configure" page is not in error state anymore, but there is still some error on the UI and in the console. So there is still something to work on there.